src/components/FormProduct/index.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+import { addProduct } from 'components/Utils/firestore.js';
+
+export const FormProduct = () => {
+  const inicialStateProduct = {
+    item: '',
+    nextPurchase: 0,
+    LastPurchasedDate: null,
+  };
+
+  const [products, setProduct] = useState(inicialStateProduct);
+
+  const [error, updateError] = useState(false);
+
+  const handleInputProduct = (e) => {
+    setProduct({
+      ...products,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  const handleSubmitProduct = (e) => {
+    e.preventDefault();
+    let { item, nextPurchase, LastPurchasedDate } = products;
```

Comment:
Another little detail, is it possible to change `LastPurchasedDate` to `lastPurchasedDate `? Same last detail, **_your work works!_** 

Now, it is your decision to change it 😊.

Reply:
Great! Thanks for your support! I appreciate it! 👍